modules/common/src/pipes/constants/language-object.const.ts

```diff
@@ -0,0 +1,39 @@
+import {LanguageObject} from '../interfaces';
+
+export const GERMAN_LANG_OBJECT: LanguageObject = {
+    full: 'de-DE',
+    locale: 'de',
+    culture: 'DE',
+};
+
+export const SWISS_LANG_OBJECT: LanguageObject = {
+    full: 'de-CH',
+    locale: 'de',
+    culture: 'CH',
+};
+
+export const AUSTRIAN_LANG_OBJECT: LanguageObject = {
+    full: 'de-AT',
+    locale: 'de',
+    culture: 'AT',
+};
+
+export const BRITISH_LANG_OBJECT: LanguageObject = {
+    full: 'en-GB',
+    locale: 'en',
+    culture: 'GB',
+};
+
+export const US_LANG_OBJECT: LanguageObject = {
+    full: 'en-US',
+    locale: 'en',
+    culture: 'US',
+};
```

Comment:
do we really need those? can't we have a list of allowed "full" langs?

from that you can already parse it out and it would make the code simpler

Reply:
you should always care more about production code... and even though, I don't agree if you would use the snapshot, you don't need those at all